### PR TITLE
Code samples update

### LINKED ISSUE #354: Update .code-samples.meilisearch.yaml
⚠️ This issue is generated. It means the examples and the namings do not necessarily correspond to the language of this repository.
Also, if you are a maintainer, feel free to add any clarification and instruction about this issue.

Sorry if this is already wholly/partially implemented. Feel free to let me know about the state of this issue in the repo.

Related to https://github.com/meilisearch/integration-guides/issues/185

----

Check the [integration-guides issue](https://github.com/meilisearch/integration-guides/issues/185) for more information and the complete description about what should be done here (each TODO item has its report in the #185).

TODO:

- [ ] New code samples regarding the new getting started
- [ ] Replace `description` into `overview` and `genre` into `genres`
- [ ] Update geosearch guide
- [ ] Remove unused code samples
- [ ] Update the primary key examples
- [ ] Add `settings_guide_sortable_1` code samples
- [ ] Add `landing_getting_started_1` code samples

### PR DESCRIPTION
**Pull request by alallema**
closed #354

### CODE REVIEW

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -311,41 +299,34 @@ settings_guide_searchable_1: |-
...
     "release_date"
   });
   client.index("movies").updateSettings(settings);
 settings_guide_sortable_1: |-
   Settings settings = new Settings();
   settings.setSortableAttributes(new String[]
   {
-    "author",
     "price",
+    "author",
```
[the remark this answers is not in the document]
Comment (alallema) [reply .code-samples.meilisearch.yaml:321]: Yes, both work but it's not consistence in every code-samples ...

[the remark this answers is not in the document]
Comment (bidoubiwa) [reply .code-samples.meilisearch.yaml:321]: Its for another time!

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -377,12 +358,88 @@ getting_started_add_documents_md: |-
...
+  client.index("movies").getTask(0);
 getting_started_search_md: |-
   ```java
   client.index("movies").search("botman");
   ```
 
   [About this SDK](https://github.com/meilisearch/meilisearch-java)
+getting_started_add_meteorites: |-
+  Path fileName = Path.of("meteorites.json");
+  String meteoritesJson = Files.readString(fileName);
```
[the remark this answers is not in the document]
Comment (bidoubiwa) [reply .code-samples.meilisearch.yaml:371]: Nevermind I saw that it was not

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -377,12 +358,88 @@ getting_started_add_documents_md: |-
...
+    new SearchRequest("").setFilter(new String[] {"_geoRadius(46.9480, 7.4474, 210000)"});
+  client.index("meteorites").search(searchRequest); 
+getting_started_geoPoint: |-
+  SearchRequest searchRequest =
+    new SearchRequest("").setSort(new String[] {"_geoPoint(48.8583701,2.2922926):asc"});
+  client.index("meteorites").search(searchRequest);
+getting_started_sorting: |-
+  SearchRequest searchRequest =
+    new SearchRequest("").setFilter(new String[] {"mass < 200"});
+    new SearchRequest("").setSort(new String[] {"mass:asc"});
```
[the remark this answers is not in the document]
Comment (alallema) [reply .code-samples.meilisearch.yaml:433]: No I don't

**Review by bidoubiwa — APPROVED** [review]
🔥🔥🔥🔥 LGTM 🔥🔥🔥🔥

**Comment by alallema** [comment]
bors merge

**Pull request merged**